src/v/pandaproxy/handlers.cc

```diff
@@ -180,4 +184,41 @@ post_topics_name(server::request_t rq, server::reply_t rp) {
       });
 }
 
+ss::future<server::reply_t>
+create_consumer(server::request_t rq, server::reply_t rp) {
+    auto req_data = ppj::rjson_parse(
+      rq.req->content.data(), ppj::create_consumer_request_handler());
+    auto group_id = kafka::group_id(rq.req->param["group_name"]);
+
+    return rq.ctx.client.create_consumer(group_id).then(
+      [group_id, rp{std::move(rp)}](kafka::member_id m_id) mutable {
+          auto adv_addr = shard_local_cfg().advertised_pandaproxy_api();
+          json::create_consumer_response res{
+            .instance_id = m_id,
+            .base_uri = fmt::format(
+              "http://{}:{}/consumers/{}",
+              adv_addr.host(),
+              adv_addr.port(),
+              m_id())};
+          auto json_rslt = ppj::rjson_serialize(res);
+          rp.rep->write_body("json", json_rslt);
+          return std::move(rp);
+      });
+}
+
+ss::future<server::reply_t>
+subscribe_consumer(server::request_t rq, server::reply_t rp) {
+    auto req_data = ppj::rjson_parse(
+      rq.req->content.data(), ppj::subscribe_consumer_request_handler());
+    auto group_id = kafka::group_id(rq.req->param["group_name"]);
+    auto member_id = kafka::member_id(rq.req->param["instance"]);
+
+    return rq.ctx.client
+      .subscribe_consumer(group_id, member_id, std::move(req_data.topics))
+      .then([group_id, rp{std::move(rp)}]() mutable {
```

Comment:
noticed that `group_id` is unused. clang-tidy plugin for editor is useful for catching these.